test/instrumentation/modules/mysql/_utils.js

```diff
@@ -1,11 +1,12 @@
 'use strict'
 
+var host = process.env.MYSQL_HOST || 'localhost'
 var mysql = require('mysql')
 
 exports.reset = reset
 
 function reset (cb) {
-  var client = mysql.createConnection({user: 'root', database: 'mysql'})
+  var client = mysql.createConnection({user: 'root', database: 'mysql', host: host})
```

Comment:
No need to specify `localhost` as the fallback as it will happen automatically if `MYSQL_HOST` isn't set, so you can just inline the environment variable to keep things together as `host` isn't used anywhere else. Potentially split the object into one line per property if you feel the line is getting too long.